tools/build_langserver/outline.txt

```diff
@@ -0,0 +1,72 @@
+LSP
+----
+
+It should work the same way as any of our other tools, such as `jarcat`.
+
+command: plz init-lsp
```

Comment:
I think we should make it `plz tool lsp`, that's normally how we get other processes. It's pretty easy to add a new one to that set (look in src/tool).

Reply:
:+1: